js/pages/results_framework/components/level_list.js

```diff
@@ -60,11 +61,16 @@ class LevelList extends React.Component {
 @inject('rootStore')
 @observer
 export class LevelListPanel  extends React.Component {
-
+    EMPTY = 0
+    NEW = 1;
+    FIRST = 2;
+    EXISTING = 3;
     constructor(props) {
         super(props);
         this.state = {
-            show_import_banner: null
+            show_import_banner: null,
+            levelsString: "",
+            level_status: this.EMPTY,
```

Comment:
A couple of comments would be helpful to indicate what these new state variables do.  They're not as self-explanatory as "show_import_bannner". Oh, and now that we've had a bit of a conversation about what you need to send to the backend, is some of this level tracking here not necessary?

Reply:
I added some comments for these status states. This is mainly for handling what buttons show above the main RF Builder panel component when first starting out building the RF Framework.

1.  Before a RF Level has been saved - Show no buttons ("Empty" or "New" status)
2. One RF Level saved - show the import indicators button and banner alert ("First" status)
3. More than one level saved - show the expand, collapse, import indicators, banner alert, and excel buttons ("Existing" status)